fix8.cmake

```diff
@@ -0,0 +1,18 @@
+# vim:ts=4:sw=4:expandtab:autoindent:
```

Comment:
I don't think we want this file to be here.

Reply:
well, 
a) since we are using this repo towards our purposes - tihs makes deploying much easier.
i dont see anything wrong with this.

its like.